packages/events/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@esri/hub-events",
+  "version": "1.3.0",
+  "description": "Module to interact with ArcGIS Hub Events in Node.js and modern browsers.",
+  "main": "dist/node/index.js",
+  "browser": "dist/umd/events.umd.js",
+  "module": "dist/esm/index.js",
+  "js:next": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@esri/arcgis-rest-auth": "^1.11.0",
+    "@esri/arcgis-rest-common-types": "^1.11.0",
+    "@esri/arcgis-rest-feature-service": "^1.11.0",
+    "@esri/arcgis-rest-items": "^1.11.0",
```

Comment:
i'd prune this list of deps to only what is currently needed (if it's not already) and re-add whatever else is needed later.